scripts/zones/Ifrits_Cauldron/mobs/Bomb_Queen.lua

```diff
@@ -20,12 +20,9 @@ end
 
 function onMobFight(mob, target)
     -- every ~20 seconds after first pet, a new pet will spawn around queen's position.
-    -- 50% chance for Prince/Princess. 1% chance for Bastard.
-
-    local mobId = mob:getID()
-
+    -- 49.75% (99/400) chance for Prince/Princess. 1% (4/400) chance for Bastard.
```

Comment:
The math below is correct but this comment is incorrect.

Should say 49.5%

Reply:
kek, its 24.75 per spawn

Reply:
you hurted my brain